src/ui/components/ErrorContent/ErrorContent.scss

```diff
@@ -32,13 +32,24 @@
         }
     }
 
+    &_size_s {
+        #{$class}__title {
+            font-size: var(--g-text-body-1-font-size);
+            line-height: var(--g-text-body-1-line-height);
+        }
+
+        #{$class}__content {
+            width: auto;
+        }
+    }
+
     &_no-controls {
         display: block;
         padding: 20px 0;
     }
 
     &__title {
-        font-weight: 500;
+        font-weight: var(--g-text-header-font-weight);
```

Comment:
we have this variable set to 500 in DC code
but isn't this is a component that DL also uses?
I don't know if they have the same override

Reply:
Good point. Will remove. Thanks